service-automation/src/com/appyschool/suiteSetup/TestSuiteSetup.java

```diff
@@ -409,40 +411,68 @@ public void testCreateCoCurricularAreas(HashMap<String, String> map) throws JSON
+		Map<String, String> branchStandardIds = CommonMethods.getBranchStandardFromMapBranchStandardSections(SuiteSetupProperties.getValue("BRANCH_ID_"+n));
 		
-		map.put("$branchId", SuiteSetupProperties.getValue("BRANCH_ID_1"));
+		map.put("$branchId", SuiteSetupProperties.getValue("BRANCH_ID_"+n));
 		map.put("$branchStandardSection", branchStandardIds.get("mapId"));
 		map.put("$standard", branchStandardIds.get("standardId"));
-
 		
 		JSONObject json = CommonUtils.getJsonFromTemplate(jsonTemplate, map);
+
 		json.put("idNumber", "Appy" + RandomStringUtils.randomNumeric(5));
 		json.put("rollNumber", RandomStringUtils.randomNumeric(5));
-		json.put("primaryPhoneNumber", RandomStringUtils.randomNumeric(5) + RandomStringUtils.randomNumeric(5));
-		json.put("alternatePhoneNumber", RandomStringUtils.randomNumeric(5) + RandomStringUtils.randomNumeric(5));
+		json.put("primaryPhoneNumber", RandomStringUtils.randomNumeric(10));
+		json.put("alternatePhoneNumber", RandomStringUtils.randomNumeric(10));
 		json.put("primaryEmailId", map.get("$firstName") + RandomStringUtils.randomNumeric(2) + "@appyschool.com");
 		json.put("secondaryEmailId", map.get("$firstName") + RandomStringUtils.randomNumeric(2) + "@appyschool.com");
 		Response response = StudentServices.createStudent(json.toString());
 		response.then().log().all();
 		
 		JSONObject responseObj = new JSONObject(response.getBody().asString());
-		verifyEquals(response.statusCode(), 200);
-		verifyTrue(responseObj.get("id") != null);
-		verifyEquals(responseObj.getString("status"), "OK");
+		SoftAssertionBase.verifyEquals(response.statusCode(), 200);
+		SoftAssertionBase.verifyEquals(responseObj.getString("status"), "OK");
 		SuiteSetupProperties.setValue("STUDENT_ID_" + n, responseObj.get("id"));
 	}
 	
+	
+		
+	 /* Create Designation
+	 * 
+	 * @param map
+	 * @throws JSONException
+	 */
+	@Test(dataProviderClass = com.appyschool.common.CommonUtils.class, dataProvider = "getDataFromFile")
+	@DataProviderArguments("./testData/configuration/createDesignation.csv")
+	public void testCreateDesignation(HashMap<String, String> map) throws JSONException {
+		
+		l++;
+		
+		String jsonTemplate = CommonUtils.getJsonTemplate("./testData/configuration/branches/json/createDesignation.json");
+		JSONObject json = CommonUtils.getJsonFromTemplate(jsonTemplate, map);
+		
+		json.put("name",RandomStringUtils.randomAlphabetic(10));
+		json.put("description",RandomStringUtils.randomAlphabetic(10));
+		
+		Response response = DesignationServices.createDesignation(json.toString());
+		response.then().log().all();
+		System.out.print(response);
+		
+		JSONObject responseObj = new JSONObject(response.getBody().asString());
+		
+		// Setting the Designation Id so that it can be used whenevevr required
+		SuiteSetupProperties.setValue("Desig_ID_" + l, responseObj.get("id"));
```

Comment:
since you are creating only one designation, so "l" variable not needed. 
Also rename variable to make them more readable.